lib/util/insightspublisher/index.js

```diff
@@ -71,7 +75,18 @@ class InsightsPublisher extends EventEmitter {
       self.removeListener('disconnected', maybeReconnect);
```

Comment:
@makarandp0 Just above this line, can you also make sure `roomSid` and `participantSid` are passed to the `reconnect()` function as well?

Reply:
thank you @manjeshbhargav for catching this!